assets/queries/terraform/gcp/stackdriver_logging_disabled/test/positive_expected_result.json

```diff
@@ -8,5 +8,10 @@
     "queryName": "Stackdriver Logging Disabled",
     "severity": "HIGH",
     "line": 13
```

Comment:
Also, update the vulnerability lines accordingly to the findings in `positive.tf` file
Please also add the  `"fileName": "positive.tf"` to the `positive_expected_result.json` items